pod/perldelta.pod

```diff
@@ -398,7 +398,7 @@ XXX
 =item *
 
 XXX Add anything here that we forgot to add, or were mistaken about, in
-the perldelta of a previous release.
+the F<perldelta> of a previous release.
```

Comment:
perldelta's are files, and each release has a file called `perldelta` by default (without version infix)